dispatch/static/manager/src/js/pages/DashboardPage.js

```diff
@@ -1,14 +1,85 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import { Link } from 'react-router'
+import { connect } from 'react-redux'
+import * as dashboardActions from '../actions/DashboardActions'
+import moment from 'moment'
```

Comment:
This is nit-picky, but I like to put all external imports before project imports. Can you move this up one line?